src/pyxplor/plot_time_series.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+
+def plot_time_series(input_df: pd.DataFrame, 
+
+                      date_column: str, 
+                      value_columns: list, 
+                      freq: str = 'D', 
+                      figsize: tuple = (10, 6),
+                      output_path: str = None, 
+                      super_title: str = "Time Series Analysis",
+                      super_title_font: int = 14) -> None:
+    """
+    Conducts exploratory data analysis on multiple time-series variables and generates line plot visualizations.
+
+    Parameters
+    ----------
+    input_df : pd.DataFrame
+        The DataFrame containing the time-series data. Expected to have at least two columns: 
+        one for dates and others for numeric values.
+        
+    date_column : str
+        The name of the column in `input_df` that contains date/time information. 
+        This column should be convertible to pandas datetime format.
+
+    value_columns : list
+        A list of names of the columns in `input_df` that contain the numeric values to be analyzed.
+
+    freq : str, optional
+        The frequency of the time-series data ('D' for daily, 'W' for weekly, etc.). Default is 'D'.
+
+    figsize : tuple, optional
+        The size of the plot in the format (width, height). Default is (10, 6).
+
+    output_path : str, optional
+        Path to save the plot. Defaults to the current working directory.
+
+    super_title : str, optional
+        Super title for the entire plot. Default is "Time Series Analysis".
+
+    super_title_font : int, optional
+        Font size for the super title. Default is 14.
+
+    Returns
+    -------
+    None
+
+    Examples
+    --------
+    # Example usage of the xplor_time_series function
+
+    # Sample DataFrame creation
+    dates = pd.date_range(start='2020-01-01', end='2020-12-31', freq='D')
+    data = pd.DataFrame({
+        'date': dates,
+        'sales': np.random.rand(len(dates)) * 100,
+        'expenses': np.random.rand(len(dates)) * 50,
+    })
+
+    # Using the function to plot 'sales' and 'expenses' time series from the DataFrame
+    xplor_time_series(data, 'date', ['sales', 'expenses'], freq='M', figsize=(12, 6))
+
+    # This will create a line plot with two lines representing 'sales' and 'expenses',
+    # aggregated monthly, and display it. The plot will not be saved to a file.
```

Comment:
I don't think we need to explain this much under example. It would be better if you explained it in the description.
```suggestion
```